libs/core/src/lib/status-indicator/status-indicator.component.scss

```diff
@@ -2148,4 +2148,7 @@
 .fd-status-indicator--xl__text {
```

Comment:
It was missed during the prev PR review, but this class makes no sense in BEM. You can't have .block--modifier__element like in your case. `.fd-status-indicator` is the block. Then it should be followed by an element, something like: `.fd-status-indicator__text` and the modifier comes last as it's modifying the default state of an element: `.fd-status-indicator__text--xl`.